README.md

```diff
@@ -623,8 +623,8 @@ See: <https://mobx.js.org/refguide/create-transformer.html>
 
 ## deepObserve
 
-Given an object, deeply observes the given object.
-It is like `observe` from mobx, but applied recursively, including all future children
+Deeply observes a given object, array or map.
+It is like `observe` from mobx, but applied recursively, including all future children.
```

Comment:
A suggestion:
Maybe change 
'''
It is like `observe` from mobx, but applied recursively, including all future children.
``' 
To 
```
Like MobX's `observe`, but applied recursively, includeing all its future children.
```

Reply:
sounds good, done